usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -100,6 +104,12 @@ def add_arguments(self, parser):
             default=CONFIG.SPARK_S3_BUCKET,
             help="The destination bucket in S3 to write the data",
         )
+        parser.add_argument(
+            "--reset-sequence",
+            action="store_true",
+            help="In the case of a Postgres sequence for the provided 'delta-table' the sequence will be reset to 1. "
+            "If the job fails for some unexpected reason then the sequence will be reset to the previous value.",
```

Comment:
Is this second line of the parameter help info directly related to the parameter? Looks like the sequence value is reset to the previous value whether this argument is used or not.

Reply:
Sequence value is only reset if there is a value for `postgres_seq_last_value`; which would only exist if it was previously changed.

Reply:
Okay gotcha. I'm following